style.css

```diff
@@ -70,14 +71,46 @@ label {
   padding: 4px 6px;
   font-size: 15px; }
 
+.text-x {
+  height: 200px;
+  min-width: 200px;
+  max-width: 200px; }
+
 .container {
-  padding-top: 50px; }
+  padding-top: 50px;
+  width: 98%; }
+
+.ld-sidebar p {
+  font-size: 17px;
+  text-align: center; }
+
+.ld-sidebar h1 {
+  text-align: center; }
+
+.choose-opt li {
+  list-style: none !important;
+  font-size: 16px;
+  padding: 3px; }
+
+.choose-opt li:hover {
+  font-size: 20px;
+  padding: 1px; }
+
+.choose-opt li i {
+  padding-right: 5px; }
 
 nav .container {
   padding-top: 0px; }
 
 .full-width {
-  width: 100%; }
+  width: 100%;
```

Comment:
Else they're able to crush the whole template with resizing the textarea. I can't stand it, if you can move textareas and crash the whole design of a website ...